test/java/com/odysseusinc/arachne/portal/front/BaseTest.java

```diff
@@ -100,38 +165,40 @@ public static void setup() {
         portalEnvs.put("spring.mail.properties.mail.smtp.auth", "false");
         portalEnvs.put("spring.mail.properties.mail.smtp.starttls.enable", "false");
         portalEnvs.put("spring.mail.properties.mail.smtp.starttls.required", "false");
-        portalEnvs.put("portal.hostsWhiteList", "localhost," + containerName);
+        portalEnvs.put("portal.hostsWhiteList", "localhost," + portalContainerName);
 
         portalEnvs.put("jasypt.encryptor.password", System.getProperty("jasypt.encryptor.password"));
 
-        portalContainer = new GenericContainer("hub.arachnenetwork.com/portal:1.10.0-SNAPSHOT")
+        portalContainer = new GenericContainer("hub.arachnenetwork.com/portal:" + VERSION)
                 .withEnv(portalEnvs)
                 .withNetwork(network)
                 .withExposedPorts(8080)
                 .waitingFor(arachneWaitStrategy);
 
         portalContainer.start();
-        String oldName = portalContainer.getContainerName().substring(1);
 
-        DockerClientFactory.instance().client().renameContainerCmd(oldName).withName(containerName).exec();
+        String oldName = portalContainer.getContainerName().substring(1);
```

Comment:
Looks like magic, not clear

substring(1);

Reply:
I can add comment :)